airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -33,10 +33,10 @@
 from typing import TYPE_CHECKING, Any, Callable, Iterable, Sequence
 
 import kubernetes
+import tenacity
 from deprecated import deprecated
-from kubernetes.client import CoreV1Api, V1Pod, models as k8s
-from kubernetes.stream import stream
-from urllib3.exceptions import HTTPError
```

Comment:
Revert it. 
Numpy checks failed because of this.

Reply:
sorry, unwanted change :)